icu4c/source/i18n/unicode/uspoof.h

```diff
@@ -359,6 +359,45 @@
  * COMMON or INHERITED, such as numbers and punctuation, are ignored when computing whether a string has multiple
  * scripts.
  *
+ * <h2>Advanced bidirectional usage</h2>
+ * If the paragraph direction with which the identifiers will be displayed is not known, there are
+ * multiple options for confusable detection depending on the circumstances.
+ *
+ * <p>
+ * In some circumstances, the only concern is confusion between identifiers displayed with the same
+ * paragraph direction.  For instance, if the identifiers are usernames prefixed with the @ symbol,
+ * that symbol will appear to the left in a left-to-right context, and to the right in a
+ * right-to-left context, so that an identifier displayed in a left-to-right context can never be
+ * confused with an identifier displayed in a right-to-left context.
+ *
+ * <p>
+ * For example, in those circumstances, the usernames "A1א" (A one aleph) and "Aא1" (A aleph 1)
```

Comment:
I find this a bit confusing, as well as repetitive.
The previous paragraph already has an example (introduced there with "For instance").
So we have two examples; the second one is in a different paragraph, which suggests a contrast with the first one, but it doesn't start with contrasting circumstances.
If these are indeed two examples for the same circumstances, then I suggest following the circumstances with one "For example" followed by a bulleted (`<ol>`) list.

Reply:
There is a sort of nesting of examples going on (we are in the example of `@`usernames, and then we are looking at specific pairs of usernames), but I agree it is illegible. Restructured, PTAL.